pkg/modules/authorization_test.go

```diff
@@ -646,7 +646,7 @@ func TestAuthorizationIngress(t *testing.T) {
 
 			return true, true, tmpCR, sourceClient, operatorConfig
 		},
-		"success - creating v1.8.0": func(*testing.T) (bool, bool, csmv1.ContainerStorageModule, ctrlClient.Client, utils.OperatorConfig) {
+		"success - creating v1.9.0": func(*testing.T) (bool, bool, csmv1.ContainerStorageModule, ctrlClient.Client, utils.OperatorConfig) {
```

Comment:
Lets leave this at `success - creating v1.8.0`. The test `success - creating` will use the cr_auth_proxy.yaml that tests with 1.9.0 as part of your changes.

Reply:
@EmilyKatdell this comment hasn't been addressed yet in your changes.